includes/documents/abstract-wcpdf-order-document.php

```diff
@@ -640,8 +640,10 @@ public function header_logo() {
 
 				if ( apply_filters( 'wpo_wcpdf_use_path', true ) && file_exists( $attachment_path ) ) {
 					$src = $attachment_path;
-				} else {
+				} elseif ( file_exists( $attachment_src ) ) {
```

Comment:
The `$attachment_src` is an URL not a local path, I think you must use this function: [wp_remote_head](https://developer.wordpress.org/reference/functions/wp_remote_head/)

The you should check if the response header code is `200`, which corresponds to success (exists).